packages/services/api/src/modules/app-deployments/providers/app-deployments-manager.ts

```diff
@@ -115,13 +137,36 @@ export class AppDeploymentsManager {
       },
     });
 
-    return await this.appDeployments.addDocumentsToAppDeployment({
+    const result = await this.appDeployments.addDocumentsToAppDeployment({
       organizationId: token.organizationId,
       projectId: token.projectId,
       targetId: token.targetId,
       appDeployment: args.appDeployment,
       operations: args.documents,
     });
+
+    const currentUser = await this.session.getViewer();
```

Comment:
create an app deployment is done via a target access token, so there is no viewer and this will reject.

Reply:
TODO @TuvalSimha: remove audit log creation for app deployments